app/views/docs/checkbox_group_view.rb

```diff
@@ -2,7 +2,7 @@
 
 class Docs::CheckboxGroupView < ApplicationView
   def view_template
-    component = "Checkbox Group"
+    component = "Checkbox"
```

Comment:
This is necessary because `component_files` helper uses component name to find component folder, and CheckboxGroup is inside checkbox folder.